schema/bom-1.5.schema.json

```diff
@@ -1192,7 +1197,7 @@
         }
       }
     },
-    "dataClassification": {
+    "dataFlow": {
```

Comment:
breaking change for people who reference the definition from downstream JSON schema documents.

Reply:
While the definition was renamed, the change is NOT breaking in that all of the service tests from v1.4 validate with v1.5. I tested against [valid-service-1.4.json](https://github.com/CycloneDX/specification/blob/master/tools/src/test/resources/1.4/valid-service-1.4.json)

Reply:
from a perspective of somebody who consumed/referenced this element this renaming is a breaking change.

if I wrote my own JSONschema, then this rename would be breaking change for me,
just if you renamed an element in the XML i were referencing. 

```json5
{
  "something":  
  { "allOf": [
    { "$ref": "cyclonedx.schema.json#/definitions/dataClassification" },
    { 
      // some extentsion or extra properties and such
    }
  ] }
}
```

Reply:
as discussed with others, i considder this as closed.
schema is consistent, breaking changes in new schema definition, old one is still untouched. 
therefore all good!